source/module_hamilt_pw/hamilt_pwdft/operator_pw/projop_pw.cpp

```diff
@@ -0,0 +1,136 @@
+#include "projop_pw.h"
+#include "module_hamilt_pw/hamilt_pwdft/radial_projection.h"
+#include "module_basis/module_nao/projgen.h"
+#include "module_basis/module_nao/atomic_radials.h"
+#include <cassert>
+#include <numeric>
+namespace hamilt
+{
+    template<typename T, typename Device>
+    void DFTU<OperatorPW<T, Device>>::read_abacus_orb(const std::string& forb,
+                                                      int& lmax,
+                                                      std::vector<int>& nzeta,
+                                                      int& nr,
+                                                      double& dr,
+                                                      std::vector<std::vector<double>>& radials)
+    {
+        std::string elem;
+        double ecut;
```

Comment:
initialize the variable

Reply:
thanks, changes were made in the latest commit.